rpcs3/Emu/Cell/Modules/sceNp2.cpp

```diff
@@ -854,12 +953,14 @@ error_code sceNpMatching2GetServerIdListLocal(SceNpMatching2ContextId ctxId, vm:
 {
 	sceNp2.todo("sceNpMatching2GetServerIdListLocal(ctxId=%d, serverId=*0x%x, serverIdNum=%d)", ctxId, serverId, serverIdNum);
 
-	if (!g_fxo->get<sce_np_2_matching_manager>()->is_initialized)
+	const auto nph = g_fxo->get<named_thread<np_handler>>();
+
+	if (!nph->is_NP2_Match2_init)
 	{
 		return SCE_NP_MATCHING2_ERROR_NOT_INITIALIZED;
 	}
 
-	return CELL_OK;
+	return not_an_error(0);
```

Comment:
What's up with this?

Reply:
ducumentation, I guess it should return a count of something if everything goes right.

Reply:
Yes it returns number of elements in server list, adding a comment.